core/board.py

```diff
@@ -2,38 +2,8 @@ class Board:
     def __init__(self):
         self.state = [[None for ele in range(10)] for index in range(10)]
 
-    def format(self):
-        formatted_board = '\n' + '    A  B  C  D  E  F  G  H  I  J'
-
-        for row in range(0, len(self.state)):
-            formatted_board += '\n'
-            if (row + 1) < 10:
-                formatted_board += ' ' + str(row + 1) + ' '
-            else:
-                formatted_board += str(row + 1) + ' '
-            for column in range(0, len(self.state[row])):
-                if self.state[column][row] is None:
-                    formatted_board += '[ ]'
-                else:
-                    formatted_board += '[X]'
-
-        formatted_board += '\n'
-        return formatted_board
-
-    def update(self, user_shot_choice):
-        letters = {
-                'A': 0,
-                'B': 1,
-                'C': 2,
-                'D': 3,
-                'E': 4,
-                'F': 5,
-                'G': 6,
-                'H': 7,
-                'I': 8,
-                'J': 9
-                }
-        numbers = {
+    def update(self, user_shot_choice, is_hit):
+        rows = {
```

Comment:
It's weird to me that these dictionaries exist within this method. I think there's probably a cleaner way to do this.